apps/sms/test/marionette/mocks/mock_navigator_moz_contacts.js

```diff
@@ -21,20 +21,33 @@ Services.obs.addObserver(function(document) {
 
   var window = document.defaultView;
 
+  function rehydratePhotos(contacts) {
```

Comment:
No one uses this functionality yet, but I want to use in the test for the [bug 1217859](https://bugzilla.mozilla.org/show_bug.cgi?id=1217859)